views/user-welcome.twig

```diff
@@ -17,6 +17,55 @@
     .tour .popover-title{
         background-color: #b2bfdc;
     }
+
+    /* Print Style Sheet - for POP reports */
+    @media print {
```

Comment:
Unfortunately I don't think this is the file we need to put it in - can we create a `print.css` file in the default theme, and then add that to the base.twig file as a CSS reference?